### ISSUE TITLE
Hasura UI breaks when browsing rows of a simple table when event trigger is present (React error 130)

### ISSUE THREAD

**Issue by MikeBosw** [hasura/graphql-engine#9144]
### Version Information

Server Version: v2.1.1
CLI Version (for CLI related issue): v2.13.0

### Environment

Docker: hasura/graphql-engine:latest

### What is the current behaviour?

For table `foobar`:

When navigating to: `http://localhost:9695/console/data/default/schema/public/tables/foobar/browse`

A full-page error view appears with the following details:

Error: Minified React error `#130`; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.

### What is the expected behaviour?

I should be able to see the rows of the table.

### How to reproduce the issue?

1. Create a table with columns id uuid and blob jsonb:
`CREATE TABLE "public"."example_table" ("id" uuid NOT NULL DEFAULT gen_random_uuid(), "blob" jsonb NOT NULL, PRIMARY KEY ("id") );`
2. Create a trigger as defined in the following table.yaml:
    ```
    - table:
        schema: public
        name: example_table
      event_triggers:
        - name: on_insert
          definition:
            enable_manual: true
            insert:
              columns: '*'
          retry_conf:
            num_retries: 0
            interval_sec: 10
            timeout_sec: 60
          webhook: http://webhook-svc/foobar
          request_transform:
            body: '{{$body.event.data.new}}'
            content_type: application/json
            template_engine: Kriti
    ```
3. Run the Hasura console and navigate to `http://localhost:9695/console/data/default/schema/public/tables/example_table/browse` in a browser

That's all I needed to do, but if that doesn't reproduce the issue, try inserting a row?

### Screenshots or Screencast

![https://ibb.co/rcGnMY2](https://i.ibb.co/CM5xvgK/hasura-browser-error.jpg)

### Please provide any traces or logs that could help here.

```
Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at oc (https://graphql-engine-cdn.hasura.io/console/assets/channel/stable/v2.1/vendor.js.gz:1:5486981)
    at https://graphql-engine-cdn.hasura.io/console/assets/channel/stable/v2.1/vendor.js.gz:1:5432151
    at Zo (https://graphql-engine-cdn.hasura.io/console/assets/channel/stable/v2.1/vendor.js.gz:1:5442287)
    at Ql (https://graphql-engine-cdn.hasura.io/console/assets/channel/stable/v2.1/vendor.js.gz:1:5482766)
    at zl (https://graphql-engine-cdn.hasura.io/console/assets/channel/stable/v2.1/vendor.js.gz:1:5473739)
    at jl (https://graphql-engine-cdn.hasura.io/console/assets/channel/stable/v2.1/vendor.js.gz:1:5473664)
    at kl (https://graphql-engine-cdn.hasura.io/console/assets/channel/stable/v2.1/vendor.js.gz:1:5470514)
    at https://graphql-engine-cdn.hasura.io/console/assets/channel/stable/v2.1/vendor.js.gz:1:5421486
    at t.unstable_runWithPriority (https://graphql-engine-cdn.hasura.io/console/assets/channel/stable/v2.1/vendor.js.gz:1:5497297)
    at oa (https://graphql-engine-cdn.hasura.io/console/assets/channel/stable/v2.1/vendor.js.gz:1:5421195)
```

### Any possible solutions/workarounds you're aware of?

I'm afraid not.

### Keywords

130

**Comment by manasag** [comment]
Hi @MikeBosw , thanks for raising this issue. 
There seems to be some issue with the CDN served UI assets with version 2.1 of Hasura. I can recommend following two ways to resolve this:
- Use newer Hasura docker image: version 2.2 or later (recommended to upgrade to latest released version which is 2.14)
- Enable serving UI assets from local, instead of CDN. Add this environment variable to docker container `HASURA_GRAPHQL_CONSOLE_ASSETS_DIR=/srv/console-assets` to enable local assets.

**Comment by tirumaraiselvan** [comment]
Closing this following the comment above. Pls reopen if the problem persists in version 2.2 or later.
